README.md

```diff
@@ -174,14 +176,16 @@ for Intel IPU ES2K.
    # git checkout ipdk_v23.07
    ```
 
-3. Build the P4 Kubernetes binaries:
-  ```bash
-  # make build
-  ```
-  Then build the Kubernetes container images:
-  ```bash
-  # make docker-build
-  ```
+3. Build the P4 Kubernetes binaries. Note that, by default, Makefile is
+   configured to build for ES2K target. To build for P4-DPDK target, use
+   "tagname=dpdk" argument to following make targets.
+   ```bash
+   # make build
+   ```
+   Then build the Kubernetes container images:
+   ```bash
+   # make docker-build
```

Comment:
And for es2k, we should also mention that use should copy the artifacts to k8s_dp/es2k before we run make docker-build.

Reply:
The copying is done by the makefile. User doesn't need to do this explicitly.

Reply:
Agree. Will make this note.

Reply:
Make already takes care of moving relevant files from the tagname target. Unless there is a bug, there is no reason to do it manually.